src/components/app/app-theme/app.theme.js

```diff
@@ -1,6 +1,6 @@
 import { createMuiTheme } from '@material-ui/core';
 import { paletteGenerator } from './app.pallete';
-import appTypography from './app.typography';
+import appTypography from './typography.styles';
```

Comment:
Rename import to `typographyStyles` because it is actually styles

Reply:
Renamed!